sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/AddressValue.java

```diff
@@ -1,106 +1,111 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
 
 package com.azure.ai.formrecognizer.documentanalysis.models;
 
-import com.azure.ai.formrecognizer.documentanalysis.implementation.util.AddressValueHelper;
 import com.azure.core.annotation.Immutable;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
-/**
- * Address field value.
- */
+/** Address field value. */
 @Immutable
 public final class AddressValue {
-
     /*
-     * Building number.
+     * House or building number.
      */
+    @JsonProperty(value = "houseNumber")
     private String houseNumber;
 
     /*
      * Post office box number.
      */
+    @JsonProperty(value = "poBox")
     private String poBox;
 
     /*
      * Street name.
      */
+    @JsonProperty(value = "road")
     private String road;
 
     /*
      * Name of city, town, village, etc.
      */
+    @JsonProperty(value = "city")
     private String city;
 
     /*
      * First-level administrative division.
      */
+    @JsonProperty(value = "state")
     private String state;
 
     /*
      * Postal code used for mail sorting.
      */
+    @JsonProperty(value = "postalCode")
     private String postalCode;
 
     /*
      * Country/region.
      */
+    @JsonProperty(value = "countryRegion")
     private String countryRegion;
 
     /*
-     * Street-level address, excluding city, state, countryRegion, and
-     * postalCode.
+     * Street-level address, excluding city, state, countryRegion, and postalCode.
      */
+    @JsonProperty(value = "streetAddress")
     private String streetAddress;
 
     /*
      * Apartment or office number
      */
+    @JsonProperty(value = "unit")
     private String unit;
 
     /*
      * Districts or boroughs within a city, such as Brooklyn in New York City or City of Westminster in London.
      */
+    @JsonProperty(value = "cityDistrict")
     private String cityDistrict;
 
     /*
      * Second-level administrative division used in certain locales.
      */
+    @JsonProperty(value = "stateDistrict")
     private String stateDistrict;
 
     /*
      * Unofficial neighborhood name, like Chinatown.
      */
+    @JsonProperty(value = "suburb")
     private String suburb;
 
     /*
-     * Building name, such as World Trade Center.
+     * Build name, such as World Trade Center.
      */
+    @JsonProperty(value = "house")
     private String house;
 
     /*
      * Floor number, such as 3F.
      */
+    @JsonProperty(value = "level")
     private String level;
 
+    /** Creates an instance of AddressValue class. */
+    private AddressValue() {}
```

Comment:
This should be public as we've already shipped this model with a public constructor.

Reply:
yep reverted this.